### ISSUE TITLE
when should i stop the training of my tensorflow model

### ISSUE THREAD

**Issue by D0o0D** [tensorflow/models#5062]
I am on win 10 - training with cpu only - sorry for this stupid question:

> when should stop training my model? meaning what the indication prove that
> my model already trained and i should stop it?

after 4 hours of training my model , now , i am at 230 steps with loss value: 0.8 to 0.5
working with faster rcnn inception v2 , is it slow?

thanks

**Comment by harshini-gadige** [comment]
@D0o0D  Usually, when the training accuracy increases but the validation accuracy starts decreasing with respect to number of iterations then we stop training a network. 
If you want in terms of error, the training can be stopped when the mean squared error turns constant or changes with a very little difference.

**Comment by ymodak** [comment]
Closing this issue since harshini-gadige's explanation gives an insight. Feel free to reopen if have any follow up questions. Thanks!
